src/transaction/log_applier_sql_log.c

```diff
@@ -612,6 +633,29 @@ sl_open_next_file (FILE * old_fp)
   return new_fp;
 }
 
+/*
+ * sl_remove_oldest_file() - remove oldest sql log file
+ *   return: error code
+ *
+ * Note:
+ *   This function is related to the ha_sql_log_keep_count system parameter.
+ *   This system parameter can be set from 2 to 5 and only that number of sql log files are kept.
+ */
+static int
+sl_remove_oldest_file (void)
+{
+  int oldest_file_id;
+  char oldest_file_path[PATH_MAX];
+
```

Comment:
```suggestion
assert (sl_Info.curr_file_id >= sql_log_keep_cnt);
```
As a pre-condition of this function.

Reply:
Not always. I will describe in the sub-task(http://jira.cubrid.org/browse/CBRD-24738).